doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -9,6 +9,19 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3349
+****************
+
+The following changes have been introduced in the v3349 of the controller:
+
+Changes
+=======
+
+* Fixed Direct Test Mode (DTM), which has been broken after version 3307
+* Fixed issue where connection parameter update could lead to disconnection
+* Fixed issue where "stream stopped" and "disconnected" events were not triggered
+* Fixed issue where controller stopped responding when a disconnection happened
```

Comment:
```suggestion
Bug fixes
=========

* Fixed the Direct Test Mode (DTM), which was broken after version 3307.
* Fixed an issue where an update to the connection parameter could lead to a disconnection.
* Fixed an issue where the "stream stopped" and "disconnected" events would not be triggered.
* Fixed an issue where the controller would stop responding when a disconnection happened.
```

Reply:
Will keep Changes header for consistency with the previous comments